doc/whats_new/v1.4.rst

```diff
@@ -345,6 +345,11 @@ Changelog
   not check for nans during input validation.
   :pr:`21807` by `Thomas Fan`_.
 
+- |API| :class:`feature_selection.SelectKBest` and
```

Comment:
Is it really API related? The API doesn’t change. Maybe ENH or something else. But no strong opinion.